common/common_methods.py

```diff
@@ -0,0 +1,7 @@
+class Common:
```

Comment:
Does this need its own namespace? Files in python act as their own namespaces, 

IE:

```
import common.common_methods as common

common.check_expected_status(foo, bar)
```